proj/hps_accel/vizier_autotune.ipynb

```diff
@@ -0,0 +1,295 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Vizier Auto-tuning\n",
+    "\n",
+    "Use this notebook to find optimal parameters for `nextpnr` using Google Vizier framework."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "###  Install python dependencies"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "! pip install -U google-api-python-client\n",
```

Comment:
nit: curious if the space suffix a convention you're following from another project? (I'm personally always have seen those without an extra space)

Reply:
I see :) thanks for the reference, I'm not sure if I personally like it but at least it's consistent :)